ui/app/templates/storage/index.hbs

```diff
@@ -0,0 +1,233 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{page-title "Storage"}}
+
+<Breadcrumb @crumb={{hash label="Storage" args=(array "storage.index")}} />{{outlet}}
+
+<StorageSubnav />
+<section class="section storage-index">
+
+<Hds::PageHeader @title="Storage" as |PH|>
+  <PH.Actions>
+    {{#if this.system.shouldShowNamespaces}}
+      <Hds::Dropdown data-test-namespace-facet as |dd|>
+        <dd.ToggleButton @text="Namespace ({{this.qpNamespace}})" @color="secondary" />
+        {{#each this.optionsNamespaces as |option|}}
+          <dd.Radio
+            name={{option.key}}
+            {{on "change" (action (mut this.qpNamespace) option.key)}}
+            checked={{eq this.qpNamespace option.key}}
+          >
+            {{option.label}}
+          </dd.Radio>
+        {{/each}}
+      </Hds::Dropdown>
+    {{/if}}
+  </PH.Actions>
+</Hds::PageHeader>
+
+  {{#if this.isForbidden}}
+    <ForbiddenMessage />
+  {{else}}
+    <Hds::Card::Container @level="base" @hasBorder={{false}} class="storage-index-table-card">
+      <header aria-label="CSI Volumes">
+        <h3>CSI Volumes</h3>
+        <p class="intro">
+          Storage configured by plugins run as Nomad jobs, with advanced features like snapshots and resizing.
+          <Hds::Link::Inline @href="https://developer.hashicorp.com/nomad/docs/concepts/plugins/csi" @icon="docs-link" @iconPosition="trailing">Read more</Hds::Link::Inline>
```

Comment:
It's not published yet, but a better link might be https://github.com/hashicorp/nomad/blob/main/website/content/docs/other-specifications/volume/csi.mdx